Doc/library/math.rst

```diff
@@ -298,9 +298,12 @@ Number-theoretic and representation functions
 
 .. function:: trunc(x)
 
-   Return the :class:`~numbers.Real` value *x* truncated to an
-   :class:`~numbers.Integral` (usually an integer). Delegates to
-   :meth:`x.__trunc__() <object.__trunc__>`.
+   Given a :class:`~numbers.Real` value *x*, return *x* with the fractional part
+   removed, leaving the integer part.  This rounds toward 0: ``trunc()`` is
+   equivalent to :func:`floor` for positive *x*, and equivalent to :func:`ceil` for
+   negative *x*. If *x* is not a float, ``trunc()`` delegates to
+   :meth:`x.__trunc__ <object.__trunc__>`, which should return an
```

Comment:
I have suggested restoring the space here and committed the change.
Also removed the parens, which are added to all function references by the sphinx config (wrongly in my opinionn, but at least it’s configurable in one place).

Reply:
In my test, :func: adds '()', :method: does not.  I like it better without.  See my separate review and suggestion about being consistent with linking.

Reply:
Devguide has `:ref:`link text <reference-label>`` with space.